src/diffusers/pipelines/unidiffuser/pipeline_unidiffuser.py

```diff
@@ -155,43 +135,34 @@ def __init__(
         # TODO: handle safety checking?
         self.safety_checker = None
 
-    # Modified from diffusers.pipelines.stable_diffusion.pipeline_stable_diffusion.StableDiffusionPipeline.enable_model_cpu_offload
-    # Add self.image_encoder, self.text_decoder to cpu_offloaded_models list
-    def enable_model_cpu_offload(self, gpu_id=0):
+    def enable_vae_slicing(self):
```

Comment:
Are we missing a "# Copied from ..." comment here?

Reply:
I have added `# Copied from` comments to the VAE slicing and tiling methods.